packages/nodes-base/nodes/Pipedrive/PipedriveTrigger.node.ts

```diff
@@ -52,14 +52,21 @@ export class PipedriveTrigger implements INodeType {
 			{
 				name: 'pipedriveApi',
 				required: true,
+				displayOptions: {
+					show: {
+						authentication: [
+							'accessToken',
+						],
+					},
+				},
 			},
 			{
-				name: 'httpBasicAuth',
```

Comment:
You removed httpBasicAuth and none. If someone has selected those when update to the new version it would break the workflow. httpBasicAuth and none has to be here and pipedriveOAuth2Api has to be added.